package ExceptionHandling.throwss;

public class Test3withThrow {
}
